fmn/api/main.py

```diff
@@ -7,12 +7,17 @@
 from pydantic import BaseModel
 
 from ..core.config import Settings, get_settings
+from ..database import init_sync_model, sync_session_maker
+from ..database.model import User
 from .auth import Identity, get_identity_optional
 
 log = logging.getLogger(__name__)
 
 app = FastAPI()
 
+init_sync_model()
```

Comment:
Nope, FastAPI is async. Check out [their doc](https://fastapi.tiangolo.com/advanced/async-sql-databases/) about databases, they also recommend using events to connect to the database.